executor/prepared.go

```diff
@@ -226,7 +226,9 @@ func (e *ExecuteExec) Build() error {
 	if err != nil {
 		return err
 	}
-	b := newExecutorBuilder(e.ctx, e.is)
+
+	b.ctx = e.ctx
+	b.is = e.is
```

Comment:
need some comments to explain this assign?

Reply:
The assign can be removed.